mailbox/event/json/src/test/java/org/apache/james/event/json/AddedSerializationTest.java

```diff
@@ -66,7 +67,7 @@
         .add("User Custom Flag")
         .build();
     private static final SortedMap<MessageUid, MessageMetaData> ADDED = ImmutableSortedMap.of(
-        MESSAGE_UID, new MessageMetaData(MESSAGE_UID, MOD_SEQ, FLAGS, SIZE, Date.from(INSTANT), MESSAGE_ID));
+        MESSAGE_UID, new MessageMetaData(MESSAGE_UID, MOD_SEQ, FLAGS, SIZE, Date.from(INSTANT), MESSAGE_ID, ThreadId.fromBaseMessageId(MESSAGE_ID)));
```

Comment:
Maybe it could be nice in serialization test to have a different value for messgeId and threadId, to show that it is supported?